validation_test.go

```diff
@@ -745,3 +749,251 @@ func TestBuiltInValidators(t *testing.T) {
 		v.Validate()
 	})
 }
+
+func TestStructWithArray(t *testing.T) {
+	type WithArray struct {
+		Extras []ExtraInfo `validate:"required|minLen:2"`
+	}
```

Comment:
'Extras []ExtraInfo' is not suitable for this method, minLen could be some other length value.

Reply:
Perhaps I missed something and therefore do not understand why it does not suitable. Could you explain? What do you mean by a different value for minLen?

Reply:
The '[]data' length is a dynamic value, and if you specify a number, it may not be suitable for real business scenarios.

Reply:
Yes, of course, this may not be suitable for some business scenarios. But these are tests that should prove that the code is working - and it is. Should I add tests for the rest of the cases?

Reply:
@fishyww  If there's no problem, I merged? 😃

Reply:
@inhere 
The code is pretty good. I'm not sure if this feature is necessary. You can merge it if you approve it.